app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt

```diff
@@ -48,4 +48,8 @@ class AuthActivity : AppCompatActivity() {
                 .replace(R.id.frame_container_auth, fragment)
                 .commit()
     }
+
+    override fun onBackPressed() {
+        loadFragment(ProfileFragment())
+    }
```

Comment:
No, this will start a whole new back stack

Reply:
@iamareebjamal Then either we have to start activity with the flag of clear_top, else we are having root view shown of the fragment in the backstack

Reply:
Make this a fragment instead

Reply:
@iamareebjamal Okay sure!